sgxs-tools/src/sgx_detect/main.rs

```diff
@@ -379,18 +392,18 @@ fn main() {
     }
     env_logger::init();
 
-    let mut support = None;
-
-    if let Some(f) = args.value_of_os("TESTFILE") {
-        support = Some(serde_yaml::from_reader(File::open(f).unwrap()).unwrap());
-    }
+    let support = if let Some(f) = args.value_of_os("TESTFILE") {
+        Some(serde_yaml::from_reader(File::open(f).unwrap()).unwrap())
+    } else {
+        None
+    };
```

Comment:
Why are you making this change? This seems completely unrelated to the issue at hand and it's not an obvious formatting issue.